app/views/support/providers/new.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <%= form_with model: [:support, @provider], local: true do |f| %>
+
+      <%= f.govuk_error_summary %>
+
+      <fieldset class="govuk-fieldset">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+          <h1 class="govuk-fieldset__heading">Add a new provider</h1>
+        </legend>
+
+
+        <%= f.govuk_text_field :provider_name, label: { text: "Provider name", size: "s" } %>
+        <%= f.govuk_text_field :provider_code, label: { text: "Provider code", size: "s" } %>
+        <%= f.govuk_collection_select :recruitment_cycle_id,
```

Comment:
I think we only want to use the current RecruitmentCycle so this field might be redundant

Reply:
Removed this as well :+1: